spec/models/govbox_set_spec.rb

```diff
@@ -167,71 +167,38 @@
           allow(affiliate).to receive(:agency).and_return(agency)
         end
 
-        it "should call Jobs.search with the query, org codes, size, hl, and lat_lon params" do
+        it "should call Jobs.search with the query, org codes, results per page, and location name params" do
           expect(Jobs).to receive(:search).
-            with(:query => 'foo', :hl => 1, :size => 10, :organization_ids => 'ABCD,BCDE', :lat_lon => '12.34,-34.56').
+            with(query: 'foo',
+                 ResultsPerPage: 10,
+                 Organization: 'ABCD,BCDE',
+                 LocationName: 'Washington, DC').
             and_return(job_openings)
-          govbox_set = GovboxSet.new('foo', affiliate, geoip_info)
-          expect(govbox_set.jobs.first.position_title).to eq('<strong>Nurse</strong>')
+          govbox_set = GovboxSet.new('foo', affiliate, location_name)
+          expect(govbox_set.jobs.first.position_title).to eq('<em>Nurse</em>')
           expect(govbox_set.modules).to include('JOBS')
         end
       end
 
       context "when the affiliate does not have a related agency with an org code" do
-        it 'should call Jobs.search with just the query, size, hl, and tags' do
-          expect(Jobs).to receive(:search).with(query: 'foo', hl: 1, size: 10, tags: 'federal').and_return nil
+        it 'should call Jobs.search with just the query, results per page' do
+          expect(Jobs).to receive(:search).with(query: 'foo', ResultsPerPage: 10).and_return nil
           GovboxSet.new('foo', affiliate, nil)
         end
       end
 
       context 'when highlighting is enabled by default' do
         it "translates '<em>' and '</em>'" do
           expect(Jobs).to receive(:search).
-            with(query: 'nursing jobs', hl: 1, size: 10, tags: 'federal').
+            with(query: 'nursing jobs', ResultsPerPage: 10).
             and_return job_openings
           govbox_set = GovboxSet.new('nursing jobs', affiliate, nil)
-          expect(govbox_set.jobs.first.position_title).to eq('<strong>Nurse</strong>')
+          expect(govbox_set.jobs.first.position_title).to eq('<em>Nurse</em>')
```

Comment:
Same q as above - I would expect this to fail, as we are not expecting anything to be highlighted.